crates/cxx-qt-gen/src/generator/structuring/mod.rs

```diff
@@ -124,6 +124,83 @@ mod tests {
         assert!(structures.is_err());
     }
 
+    #[test]
+    fn test_module_invalid_qobject() {
```

Comment:
prefer lots of small tests, so can you split these into individual ones as when they fail it's much more obvious what is causing it then
```
fn test_module_invalid_qobject_enum()
fn test_module_invalid_qobject_invokable()
fn test_module_invalid_qobject_signal()
fn test_module_invalid_qobject_inherit()
```